_resources/port1.0/group/java-1.0.tcl

```diff
@@ -34,19 +34,11 @@ pre-fetch {
         java_set_env
         # If still not present, error out
         if { ${java_version_not_found} } {
-            global os.platform os.major
-            if {${os.platform} eq "darwin" && ${os.major} >= 20} {
-                # The following check is broken on macOS 11 Big Sur so we
-                # temporarily give up on ensuring an exact Java version. See
-                # https://trac.macports.org/ticket/61445
-                ui_warn "Failed to confirm that required Java was installed; see https://trac.macports.org/ticket/61445"
-            } else {
                 ui_error "${name} requires Java ${java.version} but no such installation could be found."
                 return -code error "missing required Java version"
             }
         }
     }
```

Comment:
The indentation on these closing braces needs to be fixed.

Reply:
Done.